examples/medical-center/main.bal

```diff
@@ -15,102 +15,111 @@
 // under the License.
 
 import ballerina/io;
+import foo/medical_center.entity;
 
 public function main() returns error? {
-    MedicalItemClient miClient = check new ();
-    MedicalItem item = {
+    entity:EntityClient mcClient = check new ();
+    entity:MedicalItemInsert item = {
         itemId: 1,
         name: "item name",
-        'type: "type1",
+        itemType: "type1",
         unit: "ml"
     };
-    MedicalItem createdItem = check miClient->create(item);
-    io:println("Created item id: ", createdItem.itemId);
-
-    MedicalItem retrievedItem = check miClient->readByKey(1);
+    io:println("1");
```

Comment:
shall we remove these debug prints

Reply:
updated in [29c72ce](https://github.com/ballerina-platform/persist-tools/pull/120/commits/29c72ce16d2edf4882433055270167e57360d0f7)